src/dali/plugin/input/rest/coinbase.py

```diff
@@ -390,7 +396,7 @@ def _process_fill(
                 )
             )
         else:
-            self.__logger.debug("Unsupported transaction type (skipping): %s", json.dumps(transaction_type))
+            self.__logger.warning("Unsupported transaction type (skipping): %s", transaction_type)
```

Comment:
Should we do this (like elsewhere in the code)?
```suggestion
            self.__logger.error("Unsupported transaction type (skipping): %s. Please open an issue at %s", raw_data, self.ISSUES_URL)

```